### PR TITLE
add Performance section, review some packages

### PR DESCRIPTION
**Pull request by kamilsk**
I added new section related to performance issues.

### opentracing

- github.com repo: https://github.com/opentracing/opentracing-go
- godoc.org: https://godoc.org/github.com/opentracing/opentracing-go
- goreportcard.com: https://goreportcard.com/report/github.com/opentracing/opentracing-go
- coverage service: https://codecov.io/gh/opentracing/opentracing-go/branch/master/

### opencensus

- github.com repo: https://github.com/census-instrumentation/opencensus-go
- godoc.org: https://godoc.org/go.opencensus.io
- goreportcard.com: https://goreportcard.com/report/go.opencensus.io
- coverage service: https://github.com/avelino/awesome-go/pull/2613#issuecomment-509522793

### jaeger

- github.com repo: https://github.com/jaegertracing/jaeger
- godoc.org: https://godoc.org/github.com/jaegertracing/jaeger
- goreportcard.com: https://goreportcard.com/report/github.com/jaegertracing/jaeger
- coverage service: https://codecov.io/gh/jaegertracing/jaeger/branch/master/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1684,7 +1690,6 @@ See [go-hardware](https://github.com/rakyll/go-hardware) for a comprehensive lis
 * [go-problemdetails](https://github.com/mvmaasakkers/go-problemdetails) - Go package for working with Problem Details.
 * [go-rate](https://github.com/beefsack/go-rate) - Timed rate limiter for Go.
 * [go-sitemap-generator](https://github.com/ikeikeikeike/go-sitemap-generator) - XML Sitemap generator written in Go.
-* [go-torch](https://github.com/uber/go-torch) - Stochastic flame graph profiler for Go programs.
```
Comment (kamilsk) [inline README.md]: It's archived because flamegraph visualizations are available in go tool pprof directly.

File: README.md [hunk 2]
```diff
@@ -1724,15 +1729,13 @@ See [go-hardware](https://github.com/rakyll/go-hardware) for a comprehensive lis
 * [mssqlx](https://github.com/linxGnu/mssqlx) - Database client library, proxy for any master slave, master master structures. Lightweight and auto balancing in mind.
 * [multitick](https://github.com/VividCortex/multitick) - Multiplexor for aligned tickers.
 * [myhttp](https://github.com/inancgumus/myhttp) - Simple API to make HTTP GET requests with timeout support.
-* [netbug](https://github.com/e-dard/netbug) - Easy remote profiling of your services.
```
Comment (kamilsk) [inline README.md]: I removed it because it is too old and doesn't have any tests.

File: README.md [hunk 3]
```diff
@@ -1284,6 +1284,14 @@ See [go-hardware](https://github.com/rakyll/go-hardware) for a comprehensive lis
 * [nut](https://github.com/jingweno/nut) - Vendor Go dependencies.
 * [VenGO](https://github.com/DamnWidget/VenGO) - create and manage exportable isolated go virtual environments.
 
+## Performance
+
+* [jaeger](https://github.com/jaegertracing/jaeger) - A distributed tracing system.
+* [opencensus](https://github.com/census-instrumentation/opencensus-go) - A stats collection and distributed tracing framework.
+* [opentracing](https://github.com/opentracing/opentracing-go) - OpenTracing API for Go.
```
Comment (avelino) [inline README.md]: new packages:
- https://github.com/opentracing/opentracing-go
- https://github.com/census-instrumentation/opencensus-go
- https://github.com/jaegertracing/jaeger

pls read: https://github.com/avelino/awesome-go/blob/master/CONTRIBUTING.md#quality-standards

Comment (kamilsk) [reply README.md]: yep, I will fix that

File: README.md [hunk 4]
```diff
@@ -555,7 +556,6 @@ Please take a quick gander at the [contribution guidelines](https://github.com/a
 * [gorpc](https://github.com/valyala/gorpc) - Simple, fast and scalable RPC library for high load.
 * [grpc-go](https://github.com/grpc/grpc-go) - The Go language implementation of gRPC. HTTP/2 based RPC.
 * [hprose](https://github.com/hprose/hprose-golang) - Very newbility RPC Library, support 25+ languages now.
-* [jaeger](https://github.com/jaegertracing/jaeger) - A distributed tracing system.
```
Comment (kamilsk) [inline README.md]: @avelino, https://github.com/avelino/awesome-go/pull/2613#discussion_r301250944 - jaeger is already in the list

**Comment by kamilsk** [comment]
@avelino, I wrote [a simple script](https://github.com/kamilsk/dotfiles/commit/4d1476656fbba057ca626f32560313de13bca0f7) to check coverage for opencensus.

The result is

```
coverall
?   	go.opencensus.io	[no test files]
ok  	go.opencensus.io/exporter/stackdriver/propagation	0.014s	coverage: 79.3% of statements
ok  	go.opencensus.io/internal	0.006s	coverage: 85.7% of statements
?   	go.opencensus.io/internal/check	[no test files]
?   	go.opencensus.io/internal/readme	[no test files]
?   	go.opencensus.io/internal/tagencoding	[no test files]
?   	go.opencensus.io/internal/testpb	[no test files]
ok  	go.opencensus.io/metric	1.020s	coverage: 96.2% of statements
?   	go.opencensus.io/metric/metricdata	[no test files]
ok  	go.opencensus.io/metric/metricexport	8.521s	coverage: 96.4% of statements
ok  	go.opencensus.io/metric/metricproducer	0.006s	coverage: 100.0% of statements
ok  	go.opencensus.io/plugin/ocgrpc	0.546s	coverage: 73.7% of statements
ok  	go.opencensus.io/plugin/ochttp	1.756s	coverage: 74.8% of statements
ok  	go.opencensus.io/plugin/ochttp/propagation/b3	0.014s	coverage: 94.7% of statements
ok  	go.opencensus.io/plugin/ochttp/propagation/tracecontext	0.015s	coverage: 86.3% of statements
ok  	go.opencensus.io/resource	0.006s	coverage: 79.6% of statements
?   	go.opencensus.io/resource/resourcekeys	[no test files]
ok  	go.opencensus.io/stats	0.008s	coverage: 70.7% of statements
?   	go.opencensus.io/stats/internal	[no test files]
ok  	go.opencensus.io/stats/view	0.332s	coverage: 85.3% of statements
ok  	go.opencensus.io/tag	0.007s	coverage: 71.4% of statements
ok  	go.opencensus.io/trace	0.010s	coverage: 60.3% of statements
?   	go.opencensus.io/trace/internal	[no test files]
ok  	go.opencensus.io/trace/propagation	0.015s	coverage: 100.0% of statements
ok  	go.opencensus.io/trace/tracestate	0.007s	coverage: 100.0% of statements
ok  	go.opencensus.io/zpages	0.032s	coverage: 70.1% of statements
?   	go.opencensus.io/zpages/internal	[no test files]
total: 76.7%
```

**Comment by kamilsk** [comment]
@avelino, could you recheck, please?

**Comment by kamilsk** [comment]
@ceriath, maybe you can recheck, please?

**Pull request closed without merging**